protractor_test/index.bzl

```diff
@@ -16,8 +16,8 @@ def protractor_test(name, server, deps, extra_config = {}, data = [], enable_per
             ":%s_protractor_extra_config" % name,
             "@rules_browsers//browsers/chromium",
             "@rules_browsers//protractor_test:config",
-            Label("//:node_modules/protractor"),
-            Label("//:node_modules/tinyglobby"),
+            "//:node_modules/protractor",
+            "//:node_modules/tinyglobby",
```

Comment:
@josephperrott @devversion The presubmit does not pass with `Label` or `@rules_browsers//...`. Only this works, which means the target must be available from the ruleset consumer. This seems annoying. I believe this is because the `node_modules` directory gets linked at the root of the external workspace and a process running in the current workspace cannot see them in its ancestor tree for module resolution

Reply:
See this failing presubmit for an example: https://github.com/devversion/rules_browsers/actions/runs/16959540759/job/48068730295

Reply:
In my understanding the binary/JS would be resolved to the external repository, so it's enclosing `node_modules` would be available then. I didn't have a chance to look closely though

Reply:
One thing that seems particularly annoying about this: you just changed this @josephperrott. It seems that WORKSPACE handles this and Bzlmod cannot. At least I presume that's why you changed it. That makes it difficult to do this right for different consumers.

Reply:
Yeah, passing. Ready for review.